samples/sensor/mcp9600/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.13.1)
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+project(lm73c)
```

Comment:
Rename the project to mcp9600

Reply:
fixed, will do a new PR separate and close this one.

Reply:
Why?

Reply:
Just split the commits locally, and force-push an updated branch:

    git push r.whatever +sensor_mcp9600

Reply:
The separate PR is for the request below to split the driver and sample in to two commits, but my git knowledge may be lacking (use it a lot, but this is my first time adding to a large project like this)

Reply:
See the [Contribution Workflow](https://docs.zephyrproject.org/latest/contribute/index.html#contribution-workflow) for some git tips.